pajbot/modules/vanish.py

```diff
@@ -19,7 +19,7 @@ class VanishModule(BaseModule):
             label="Command name (i.e. vanish)",
             type="text",
             required=True,
-            placeholder="Command name (no !)",
```

Comment:
Any particular reason why you changed this? I assume in preparation for #697?

Reply:
Yeah, but is also just seemed to sound better